src/NumericalAlgorithms/DiscontinuousGalerkin/Formulation.hpp

```diff
@@ -0,0 +1,23 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <iosfwd>
+
+namespace dg {
+/*!
+ * \ingroup DiscontinuousGalerkinGroup
+ * \brief The DG formulation to use
+ *
+ * - The `StrongInertial` formulation is also known as the integrate then
+ *   transform, where the integration is done over the physical/inertial frame
```

Comment:
- "known as the integrate then transform" -> "known as the integrate then transform formulation" or alternatively "know as "integrate then transform"" (I favor the first option)
- "is done over the physical/inertial frame" -> "is done over the physical/inertial coordinates" or "is done in the physical/inertial frame"
- after "frame" (or whatever replacement), add a comma. I think the "uses the strong form" is a statement about the formulation, not the integration, so it shouldn't be in the integration clause.

Reply:
Also, how about you `\cite Teukolsky2015ega` since that's where "integrate then transform" is explained